test/metabase/api/permissions_test.clj

```diff
@@ -156,6 +156,24 @@
     (testing "make sure a non-admin cannot fetch the perms graph from the API"
       (mt/user-http-request :rasta :get 403 "permissions/graph"))))
 
+(deftest fetch-perms-graph-by-group-id-test
+  (testing "GET /api/permissions/graph"
+    (testing "make sure we can fetch the perms graph from the API"
+      (t2.with-temp/with-temp [PermissionsGroup {group-id :id :as group}    {}
+                               Database         db                          {}]
+        (perms/grant-permissions! group (perms/data-perms-path db))
+        (let [graph (mt/user-http-request :crowberto :get 200 (format "permissions/graph/group/%s" group-id))]
+          (is (= #{group-id} (set (keys graph)))))))))
```

Comment:
Maybe slightly better to assert on the shape of the entire result graph, to make sure that the queries to fetch the perms are working as intended? Same for the other tests.

Reply:
Good point: We have the schemas laying around anyway!